### PR TITLE
Cluster Time Resource

### PR DESCRIPTION
**Pull request by Krunal-Thakkar**
# Description
A few sentences describing the overall goals of the pull request's commits.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- Test Pull Request

##### OUTPUT

```paste below

```
##### ADDITIONAL INFORMATION

```paste below

```
# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [x] I have maintained at least 80% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have maintained backward compatibility

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [ ] Acceptance tests

image

### CODE REVIEW

**Review by Krishnan-Priyanshu — APPROVED** [review]

**Comment by Krunal-Thakkar** [comment]
run e2e test TestAccClusterTime

File: powerscale/provider/cluster_time_resource.go [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+// Metadata describes the resource arguments.
+func (r *ClusterTimeResource) Metadata(ctx context.Context, req resource.MetadataRequest, resp *resource.MetadataResponse) {
+	resp.TypeName = req.ProviderTypeName + "_cluster_time"
+}
+
+// Schema describes the resource arguments.
+func (r *ClusterTimeResource) Schema(ctx context.Context, req resource.SchemaRequest, resp *resource.SchemaResponse) {
+	resp.Schema = schema.Schema{
+		MarkdownDescription: "This resource is used to manage the Cluster Time Settings entity of PowerScale Array. " +
+			"PowerScale Cluster Time Settings provide the ability to configure owner settings on the cluster." +
```
Comment (shenda1) [inline powerscale/provider/cluster_time_resource.go:63]: owner -> time

File: powerscale/provider/cluster_time_resource.go [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+			"We can Create, Update and Delete the Cluster Time Settings using this resource. We can also import existing Cluster Time Settings from PowerScale array. " +
+			"Note that, Cluster Time Settings is the native functionality of PowerScale. When creating the resource, we actually load Cluster Time Settings from PowerScale to the resource state. ",
+		Description: "This resource is used to manage the Cluster Time Settings entity of PowerScale Array. " +
+			"PowerScale Cluster Time Settings provide the ability to configure Time settings on the cluster." +
+			"We can Create, Update and Delete the Cluster Time Settings using this resource. We can also import existing Cluster Time Settings from PowerScale array. " +
+			"Note that, Cluster Time Settings is the native functionality of PowerScale. When creating the resource, we actually load Cluster Time Settings from PowerScale to the resource state. ",
+		Attributes: map[string]schema.Attribute{
+			"id": schema.StringAttribute{
```
Comment (shenda1) [inline powerscale/provider/cluster_time_resource.go:71]: Please check if the description is matching with the value getting stored

Comment (Krunal-Thakkar) [reply powerscale/provider/cluster_time_resource.go:65]: Updated

**Comment by Krunal-Thakkar** [comment]
run e2e test TestAccClusterTime

**Review by shenda1 — APPROVED** [review]

**Comment by Krunal-Thakkar** [comment]
run e2e test TestAccClusterTime

**Comment by shenda1** [comment]
@Krunal-Thakkar Please check e2e

**Comment by Krunal-Thakkar** [comment]
run e2e test TestAccClusterTimeResource

**Comment by Krunal-Thakkar** [comment]
run e2e test ClusterTimeResource

**Pull request merged**